docs/source/install.rst

```diff
@@ -129,6 +129,9 @@ together with all its dependencies into it:
     $ sudo virtualenv /opt/peekaboo
     $ sudo /opt/peekaboo/bin/pip install .
 
+**Note**: Do not use ``./setup.py install`` for installation since there are
+bugs in setuptools which make dependency resolution fail sometimes.
```

Comment:
Do you want to mention `pyasn1` here?

Reply:
Yes, normally I'm a fan of giving reasons and context as well. But in this case I consciously decided against it as to not bloat the install doc too much. Also, with our requirements changing, the actual module hitting the problem might shift away from `pyasn1/pyasn1-modules` to something else.

We have an extensive explanation linked from the issue closed by the commit and the commit message gives that link. Users interested in the context could do a `git blame` to find the `Closes #80` in the comment and then read the background there. I can't think of much to make this easier to access apart from maybe an entry in the changelog.